website/src/contribute/committer-guide.md

```diff
@@ -100,3 +128,6 @@ Instead, pull it all into the subject line:
     Merge pull request #1234: [BEAM-7873] Fix the foo bizzle bazzle
 
 If you have comments to add, put them in the body of the commit message.
+
+After merging the pull request, the committer is responsible for resolving associated JIRA(s)
```

Comment:
Do you mean the contributor's responsibility, or the JIRA assignee? The contributor has the most context on whether the PR completes the associated JIRA. It might be that a JIRA requires multiple PR's (or something external, like a release) to complete.

I think it would be difficult for the committer to take this responsibility.

Reply:
Yea, I think it should be the contributor who did the change and presumably has the Jira assigned to them. We do have a problem with Jira's staying open until they get re-triaged but not sure that fits with the discussion we had on list.

Reply:
It's a trade off between full context of a JIRA and a well managed JIRA task list. Contributors could have more context on a JIRA but contributors could leave the JIRA open unintentionally. 

A compromise could be that committer asks in a JIRA if it can be close after approve the PR, and close the JIRA if no reply shows up within a reasonable time (like 3 days). Contributor could always reopen the JIRA or add more context later.

Reply:
And yes, seems like this point is not on the discussion thread. We might need to have a discussion for it since it affects both roles (contributor and committer).

Reply:
We should move this line out of the PR and discuss it separately. To review a PR, I probably need to sufficiently understand the scope of the JIRA and what the change I just merge addresses or not. Typically the committer merging the PR takes the last action in the chain and the contributor may not return just to close out the JIRA. The committer is also more likely to be aware of the versioning.

Reply:
I'm also in favor of removing this line. I think it's nice to do (especially if it's obvious and you have the context in your head) but not essential.

Reply:
Removed.